### DISCUSSION TITLE
How to add animation  (opacity ease-in-easeout) to a grouped rectangles?

### DISCUSSION THREAD

**Discussion by alamenai** [fabricjs/fabric.js#9761] [Q&A]
Hi,

I have a group of elements ( rectangles ) and I want to add some sort of animation ( opacity 0 - 1 ) for a duration of  2 seconds.

## Code

```
   group.forEachObject((object) => {

      canvas.add(object);
      object.setCoords();

// How to add animation here ?

    });
```

**Comment by asturur** [comment]
I m sorry, what did you try and what is not working for you?
